lib/prom_ex/plugins/ecto.ex

```diff
@@ -68,6 +68,11 @@ if Code.ensure_loaded?(Ecto) do
       ]
     end
 
+    @doc false
+    def handle_event(_event_name, event_measurement, event_metadata, _config) do
```

Comment:
There could be other proxy events that need to be handled so let's perhaps give this a more descriptive name: `handle_proxy_query_event` ?